tests/integration/dataframe/test_dataframe.py

```diff
@@ -20,7 +30,7 @@ def create_table(client: DataFrameClient):
     tables = []
 
     def _create_table(table: models.CreateTableRequest) -> str:
```

Comment:
Given the `or basic_table_model` below, did you intend to make `table` optional?

```suggestion
    def _create_table(table: Optional[models.CreateTableRequest] = None) -> str:
```

(with an import being required)